### ISSUE TITLE
Discuss generating contribution

### ISSUE THREAD

**Issue by anproulx** [osumontreal/OLS-2#19]
Discussion on how do we generate contribution from interest for people landing on the OSUM page

1. Include information regarding :
- Where are we currently?
- Where are we going?
- How to help in landing ? (Helped wanted)

2. Having good first issues linked where we can immediatly assess : 
- Summary of issues
- Decision (if one has been taken)

### Decision

**Comment by samuelorion** [comment]
Yes, I think this is a key thing to work on (I imagine you guys have already done some work on this!). Key ideas from our meeting where 
> picture yourself as a new member - you land on the page, how do we generate contribution from interest (so … landing page, where are we currently, where are we going - have (how to help in landing (HELP WANTED) (with good first issues linked)))
